apps/web/pages/404.tsx

```diff
@@ -42,6 +42,10 @@ export default function Custom404() {
   const isSubpage = router.asPath.includes("/", 2) || isSuccessPage;
   const isSignup = router.asPath.startsWith("/signup");
   const isCalcom = process.env.NEXT_PUBLIC_WEBAPP_URL === "https://app.cal.com";
+
+  // In case of invalid email verification token, we intentionally redirect to 404 from the API
+  const isInvalidToken = router.asPath === "/404";
```

Comment:
@sean-brydon @Udit-takkar this is causing other 404 pages to show as invalid token when it's not the case. Will revert this PR.